test/test_tags.rb

```diff
@@ -747,6 +748,7 @@ def highlight_block_with_opts(options_string)
     context "with symlink'd include" do
 
       should "not allow symlink includes" do
+        Dir.mkdir("tmp") unless File.exists?("tmp")
```

Comment:
I thought about using `File.directory?` but settled on `File.exists?` in case tmp exists but isn't a directory. Perhaps that's not a good enough reason, as it will fail on the next line anyway.